docs/radar.js

```diff
@@ -252,7 +252,7 @@ function radar_visualization(config) {
         .attr("text-anchor", "middle")
         .style("fill", config.rings[i].color)
         .style("opacity", 0.35)
-        .style("font-family", "Arial, Helvetica")
+        .style("font-family", config.font_family)
```

Comment:
As implemented right now, the `font_family` parameter is considered as mandatory, which is a breaking change. Can you read it at the beginning of the function as a variable and provide `"Arial, Helvetica"` as default value? We did the same for the `scale` parameter.

Reply:
Added, thanks a lot for the pointer!